rkovalenko/CircleOfChainedWords.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace TestConsoleApp
+{
+    public static class CircleOfChainedWords
+    {
+        public static bool IsLooped(List<string> listOfWords)
+        {
+            if(!listOfWords.Any() || listOfWords.Count == 1)
+            {
+                return false;
+            }
+
+            var startingWordIndex = 0;
+            var count = listOfWords.Count;
+            for(var i = 1; i < count;)
+            {
+                // compare last char of one word and the first char of the second word
+                if (listOfWords[startingWordIndex][^1] == listOfWords[i][0])
+                {
+                    if(i - startingWordIndex > 1)
+                    {
+                        var overlappingIndex = (startingWordIndex + 1) % count;
```

Comment:
Index arithmetic and overlaps can benefit from some more explanatory comments ;)